src/admin/shared/helpers/filters.ts

```diff
@@ -128,15 +128,27 @@ export function getMultiOptionsFilters(
 
 						// selected values => referenceTable.props in selected values array
 						...without(filterValues, NULL_FILTER).map((value: string) => {
+							if (keyIn) {
+								if (labelPath) {
+									return {
+										[referenceTable]: {
+											[labelPath]: { _in: value },
+										},
+									};
+								}
+								return {
+									[referenceTable]: { _in: value },
+								};
+							}
 							if (labelPath) {
 								return {
 									[referenceTable]: {
-										[labelPath]: { _has_key: value },
+										[labelPath]: { _has_keys_any: value },
```

Comment:
This changes the behavior for the existing filters:

before it was an "AND" operation, now it is an "OR" operation.
For organisation this makes sense, but for subjects it might not.
You probably want to make this an option in the getMultiOptionsFilter function: operator: 'AND' | 'OR'

filter subjects on production: (AND)
![image](https://user-images.githubusercontent.com/1710840/125962393-6ddb901f-ae91-4ee3-86dc-325d0cda5fd0.png)

filter subjects in this PR: (OR)
![image](https://user-images.githubusercontent.com/1710840/125962416-72122aae-1eb1-4d75-b854-24c4c2357781.png)

Reply:
checked - both subjects and organizations behave as OR filters now